otter/integration/tests/test_convergence.py

```diff
@@ -689,12 +747,11 @@ def test_group_config_update_triggers_convergence(self):
         )
 
 
-class ConvergenceSet1WithCLB(ConvergenceSet1):
+class ConvergenceSet1WithCLB(unittest.TestCase):
```

Comment:
I was going to rename this one ConvergenceWith1CLB in the next PR - can do this now :D